app.spec.js

```diff
@@ -0,0 +1,139 @@
+import request from 'supertest';
+import '@babel/polyfill';
+import app from './app';
+import shortid from 'shortid'; 
+
+describe('api', () => {
+    let notes;
+    beforeEach(() => {
+        notes = [
+            {
+                id: 1,
+                title: "Trapper Keeper",
+                issues: [
+                {
+                    id: 21,
+                    body: "Finish project",
+                    completed: false
+                },
+                {
+                    id: 22,
+                    body: "Start project",
+                    completed: false
+                },
+                {
+                    id: 23,
+                    body: "Test project",
+                    completed: false
+                },
+                {
+                    id: 24,
+                    body: "Deploy to Heroku",
+                    completed: false
+                }
+                ]
+            },
+            {
+                id: 2,
+                title: "Hello",
+                issues: [{id: 25, body: "beep", completed: true}],
+            }
+        ];
+        app.locals.notes = notes;
+    });
+
+    describe('get /api/v1/notes', () => {
+        it('should return a 200', async () => {
+            const response = await request(app).get('/api/v1/notes');
+            expect(response.status).toBe(200);
+        });
+
+        it('should respond with an array of notes', async () => {
+            const response = await request(app).get('/api/v1/notes');
+            expect(response.body).toEqual(notes);
+        });
+
+        it('should return a 200 if the note exists', async () => {
```

Comment:
I think these last 4 tests should be in a separate describe block since they are making a get call to a different endpoint path